server/routes/api/team.ts

```diff
@@ -60,4 +62,65 @@ router.post("team.update", auth(), async (ctx) => {
   };
 });
 
+router.post("team.create", auth(), async (ctx) => {
+  const { user } = ctx.state;
+  const { name } = ctx.body;
+  assertLength(name, 2, "Name must be 2 or more characters");
+
+  const existingTeam = await Team.scope("withAuthenticationProviders").findByPk(
+    user.teamId
+  );
+
+  authorize(user, "createTeam", existingTeam);
+
+  const authenticationProviders = existingTeam?.authenticationProviders.map(
+    (provider) => {
+      return {
+        name: provider.name,
+        providerId: provider.providerId,
+      };
+    }
+  );
+
+  invariant(
+    authenticationProviders?.length,
+    "authentication provideers must exist"
+  );
+
+  const team = await teamCreator({
+    name,
+    subdomain: name,
+    authenticationProviders,
+    ip: ctx.ip,
+  });
+
+  const newUser = await User.create({
+    teamId: team.id,
+    name: user.name,
+    email: user.email,
+    service: null,
```

Comment:
```suggestion
```

Shouldn't need this? This column is hella deprecated

Reply:
This field is used to populate the cookie that says "last time you signed in with Google/Slack/Email/Microsoft". What I should have done is propagate that cookie. Will update